builddefs/bootloader.mk

```diff
@@ -199,6 +199,11 @@ endif
 ifeq ($(strip $(BOOTLOADER)), tinyuf2)
     OPT_DEFS += -DBOOTLOADER_TINYUF2
     BOOTLOADER_TYPE = tinyuf2
+    FIRMWARE_FORMAT = uf2
+
+    EEPROM_DRIVER ?= wear_leveling
+    WEAR_LEVELING_DRIVER ?= legacy
+    OPT_DEFS += -DFEE_PAGE_BASE_ADDRESS=0x08008000
```

Comment:
This looks really specific to STM32F4x1; do we really want to specify these options here and not in `mcu_selection.mk` (or even in `common_features.mk`, although the code in there is probably already too complex)?

Also `-DFEE_PAGE_BASE_ADDRESS=0x08008000` affects only the `vendor` driver, and the `legacy` wear leveling driver already handles the `defined(BOOTLOADER_TINYUF2)` case for F4x1 (and, BTW, with a different offset — +48K instead of +32K). And this value is applied unconditionally, unlike other assignments (not sure what could be done about that though, apart from doing that in some `*.h` file, if you really want to support the `EEPROM_DRIVER=vendor` configuration too).

Reply:
mcu_selection is also the wrong place, the remaining linker config is on my list of things to remove. Mixtures of config is not high up on my list of priorities, as mostly the end user just wants a configuration that works, and we want easier to maintain keyboard files.

We can worry about non F4x1 boards when someone needs that support, as its going to involve other changes anyway.